INSTALL.md

```diff
@@ -11,9 +11,24 @@
 
     Examples: `/home/myuser/www/web/` or `/var/sites/<project-name>/web/`
 
-## Clean install
+## Prerequisite
 
-1. You can get eZ Platform using composer with the following commands:
+  These instructions assumes you have strong technical knowledge and have already installed PHP, web server & a database server with a corresponding clean database needed for this software.
+  For further information see: https://doc.ez.no/display/EZP/Requirements
+
+## Install
+
+1. **Get eZ Platform**:
+
+       A. **Archive** (tar/zip)
+
+       - Extract the eZ Platform 2015.01 or higher archive to a directory (herby refered to as `<root-dir>`).
+       - Modify ezpublish/config/parameters.yml to reflect your database name/server
```

Comment:
That's what I meant by "differs".

I know it used to be done that way (having parameters.yml created when building), but on the other hand, we weren't using it for anything critical for runtime.

On the other hand, I'm not sure what's better: asking to edit the file, or asking to run composer to generate it.